lldb/source/Commands/CommandObjectMultiword.cpp

```diff
@@ -429,11 +428,12 @@ llvm::StringRef CommandObjectProxy::GetUnsupportedError() {
   return "command is not implemented";
 }
 
-bool CommandObjectProxy::Execute(const char *args_string,
+void CommandObjectProxy::Execute(const char *args_string,
                                  CommandReturnObject &result) {
   CommandObject *proxy_command = GetProxyCommandObject();
-  if (proxy_command)
-    return proxy_command->Execute(args_string, result);
+  if (proxy_command) {
+    proxy_command->Execute(args_string, result);
```

Comment:
seems there should be a `return` after this line, which I presume you meant to add, since you added parens.

Reply:
Agreed and fixed! Thanks for catching that.